### ISSUE TITLE
Plugin does not load

### ISSUE THREAD

**Issue by ghost** [punteroo/TF2-Item-Plugins#13]
**Describe the bug**
Cannot get the plugin to load from a clean restart on version 3.1.0. It will work if you reload the plugin while the server is already running, but otherwise does not load by itself.

**To Reproduce**
1. Install plugin
2. Restart server
3. Plugin fails to load

```
L 11/02/2021 - 20:34:18: [SM] Exception reported: Invalid convar handle 0 (error 4)
L 11/02/2021 - 20:34:18: [SM] Blaming: tf2item_weapons.smx
L 11/02/2021 - 20:34:18: [SM] Call stack trace:
L 11/02/2021 - 20:34:18: [SM]   [0] ConVar.BoolValue.get
L 11/02/2021 - 20:34:18: [SM]   [1] Line 106, tf2item_weapons.sp::OnPluginStart
L 11/02/2021 - 20:34:18: [SM] Unable to load plugin "tf2item_weapons.smx": Error detected in plugin startup (see error logs)
L 11/02/2021 - 20:34:18: [SM] Exception reported: Invalid convar handle 0 (error 4)
L 11/02/2021 - 20:34:18: [SM] Blaming: tf2item_cosmetics.smx
L 11/02/2021 - 20:34:18: [SM] Call stack trace:
L 11/02/2021 - 20:34:18: [SM]   [0] ConVar.BoolValue.get
L 11/02/2021 - 20:34:18: [SM]   [1] Line 103, tf2item_cosmetics.sp::OnPluginStart
L 11/02/2021 - 20:34:18: [SM] Unable to load plugin "tf2item_cosmetics.smx": Error detected in plugin startup (see error logs)
```

**Comment by punteroo** [comment]
Oops! Overlooked the ConVar declaration for first loads (since I focused entirely in Late Loading)

Issuing v3.1.1 right now as we speak!
Thank you so much for the heads up on this!

**Comment by punteroo** [comment]
This was fixed in v3.1.1: https://github.com/punteroo/TF2-Item-Plugins/releases/tag/v3.1.1

Thank you!
